lib/puppet/module.rb

```diff
@@ -56,7 +56,7 @@ def self.is_module_namespaced_name?(name)
   attr_writer :environment
 
   attr_accessor :dependencies, :forge_name
-  attr_accessor :source, :author, :version, :license, :puppetversion, :summary, :description, :project_page
+  attr_accessor :source, :author, :version, :license, :summary, :description, :project_page
```

Comment:
Removing the accessor is a breaking change. I think you want to add explicit getter/setters that return nil. May want to issue a deprecation warning (though I could be convinced to make it a docs only deprecation).